documentation/building/official-build-instructions.md

```diff
@@ -5,20 +5,20 @@ WARNING: These instructions will only work internally at Microsoft.
 
 To kick off an official build, go to this build definition: https://dev.azure.com/dnceng/internal/_build?definitionId=528.
 
-This signs and publishes the following packages to the dotnet feed (currently https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet5/nuget/v3/index.json):
+This signs and publishes the following packages to the tools feed (https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json):
  - dotnet-dump
+ - dotnet-gcdump
  - dotnet-sos
  - dotnet-trace
  - dotnet-counters
  - dotnet-monitor
- - Microsoft.Diagnostic.TestHelpers
```

Comment:
Doesn't this get published for the dotnet-diagnostics-test repo? The csproj has IsPackable set to true, but IsShipping set to false so I'm not sure.

Reply:
It gets published. But we don't release it to nuget.

Reply:
But this section isn't talking about nuget but the tools feed.

Reply:
Never mind.  This doc is about what gets published to the nuget feed.